### ISSUE TITLE
Receive message timeout for subscriptions in Ktor composite adapter

### ISSUE THREAD

**Issue by ermadmi78** [ermadmi78/kobby#23]
The [GraphQL over WebSocket Protocol](https://github.com/apollographql/subscriptions-transport-ws/blob/master/PROTOCOL.md) supports **GQL_CONNECTION_KEEP_ALIVE** messages that the server must periodically send in order to keep websocket connection alive. The generated Ktor composite adapter just ignores these messages now. But if the server stops sending any messages, then the subscription event listener will simply freeze. 
We need to add the ability to set a timeout for receiving subscribed messages in the generated adapter to take full advantage of the GraphQL over WebSocket protocol.

**Comment by ermadmi78** [comment]
Now generated Ktor composite adapter has additional property `subscriptionReceiveTimeoutMillis` - the receive message timeout in milliseconds for subscriptions:
```kotlin
public open class CinemaCompositeKtorAdapter(
  protected val client: HttpClient,
  protected val httpUrl: String,
  protected val webSocketUrl: String,
  protected val mapper: CinemaMapper,
  protected val requestHeaders: Map<String, String> = mapOf(),
  protected val subscriptionPayload: Map<String, Any?>? = null,
  protected val subscriptionReceiveTimeoutMillis: Long? = null,    // <= configure receive timeout here
  protected val httpAuthorizationTokenHeader: String = "Authorization",
  protected val webSocketAuthorizationTokenHeader: String = "authToken",
  protected val idGenerator: () -> String = { Random.nextLong().toString() },
  protected val listener: (CinemaRequest) -> Unit = {}
) : CinemaAdapter {
  ...
}
```

By default, `subscriptionReceiveTimeoutMillis` is null and does not apply to subscriptions. 

You can pass receive timeout to constructor:
```kotlin
private fun createKtorAdapter(): CinemaAdapter {
    val client = HttpClient {
        install(WebSockets)
    }
    val mapper = jacksonObjectMapper()
        .registerModule(ParameterNamesModule(JsonCreator.Mode.PROPERTIES))

    return CinemaCompositeKtorAdapter(
        client,
        "http://localhost:8080/graphql",
        "ws://localhost:8080/subscriptions",
        object : CinemaMapper {
            override fun serialize(value: Any): String =
                mapper.writeValueAsString(value)

            override fun <T : Any> deserialize(content: String, contentType: KClass<T>): T =
                mapper.readValue(content, contentType.java)
        },
        subscriptionReceiveTimeoutMillis = 23000
    )
}
```

Or ask Kobby to generate default value for `subscriptionReceiveTimeoutMillis`.

**Gradle**:
```kotlin
plugins {
    kotlin("jvm")
    id("io.github.ermadmi78.kobby") version "1.5.0"
}

kobby {
    kotlin {
        adapter {
            ktor {
                receiveTimeoutMillis = 23000
            }
        }
    }
}
```

**Maven**:
```xml
<build>
    <plugins>
        <plugin>
            <groupId>io.github.ermadmi78</groupId>
            <artifactId>kobby-maven-plugin</artifactId>
            <version>1.5.0</version>
            <executions>
                <execution>
                    <phase>generate-sources</phase>
                    <goals>
                        <goal>generate-kotlin</goal>
                    </goals>
                    <configuration>
                        <kotlin>
                            <adapter>
                                <ktor>
                                    <receiveTimeoutMillis>23000</receiveTimeoutMillis>
                                </ktor>
                            </adapter>
                        </kotlin>
                    </configuration>
                </execution>
            </executions>
        </plugin>
    </plugins>
</build>
```

**Comment by ermadmi78** [comment]
Available since release 1.5.1
